secrets.py

```diff
@@ -1,9 +1,11 @@
 # Secret credentials for the robot. Ask us on Slack if you want a copy!
+# If your repo is dirty after updating this file, run:
+#     git update-index --skip-worktree secrets.py
```

Comment:
I've added in this comment, which I never remember on my own. However, I worry that pulling this change into your own repo might blow away the secrets you've got. If that's a problem, I can send them to you again over Slack.